ksqldb-parser/src/main/antlr4/io/confluent/ksql/parser/SqlBase.g4

```diff
@@ -223,9 +223,10 @@ values
     ;
 
 selectItem
-    : expression (AS? identifier)?  #selectSingle
-    | identifier '.' ASTERISK       #selectAll
-    | ASTERISK                      #selectAll
+    : expression (AS? identifier)?                       #selectSingle
+    | base=primaryExpression STRUCT_FIELD_REF ASTERISK   #selectStructAll
```

Comment:
Ah.... for when we try and catch all the cases... 

https://github.com/confluentinc/ksql/blob/master/ksqldb-parser/src/main/antlr4/io/confluent/ksql/parser/SqlBase.g4#L312-L328